README.md

```diff
@@ -71,6 +71,16 @@ Then, render it in your views with the `component` helper (or its alias `c`).
 = c "button"
 ```
 
+Make sure to include javascript pack tag in your application layout file, for instance:
+
+```slim
+/ app/views/layouts/application.html.erb
+
+= javascript_pack_tag "application"
```

Comment:
A full example with `stylesheet_pack_tag` in the head of your layout, and `javascript_pack_tag` before the closing `body` tag should be better here.